pages/clusters/overview.md

```diff
@@ -1,6 +1,9 @@
 # Clusters overview
 
-Clusters are a new way of managing your Buildkite agents. They allow teams to self-manage their agent pools, let admins create isolated sets of agents and pipelines within the one Buildkite organization, and help to make agents and queues more discoverable across your organization.
+>📘 Clusters
+> Clusters will be enabled for all organizations on February 26 2024.
+
+Clusters are a new way of managing your Buildkite agents. They allow teams to self-manage their agent pools, let admins create isolated sets of agents and pipelines within the one Buildkite organization, help to make agents and queues more discoverable across your organization, and provide easily accessible queue metrics.
```

Comment:
Suggest streamling this sentence a tad. Since clusters has been out for a while, I'd recommend removing the first sentence as the real value statements start from the 2nd sentence.

```suggestion
Clusters allow teams to self-manage their Buildkite agent pools, admins to create isolated sets of agents and pipelines within the one Buildkite organization, helps make agents and queues more discoverable across your organization, and provides easily accessible queue metrics.
```